CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ And please only add new entries to the top of this list, right below the `# Unre
 
 # Unreleased
 
+- **Breaking:** Bump MSRV from `1.57` to `1.60`.
```

Comment:
Note that `objc2` doesn't have a MSRV policy yet, see https://github.com/madsmtm/objc2/issues/203.

But since we use a pinned dependency, this shouldn't be a problem (yet).